draft-ietf-ppm-dap.md

```diff
@@ -1877,11 +1910,6 @@ Since VDAF preparation completes in a constant number of rounds, it will never
 be the case that preparation is complete for some of the reports in an
 aggregation job but not others.
 
-If the Leader fails to process the response from the Helper, for example because
-of a transient failure such as a network connection failure or process crash,
-the Leader SHOULD re-send the original request unmodified in order to attempt
-recovery (see {{aggregation-step-skew-recovery}}).
```

Comment:
I think this is what @ekr was referring to in #695, which called out S 4.6.2.1. In any case, we don't need this paragraph: the method for agg job init is PUT, which is safe and idempotent per RFC 9110, so we don't need to re-explain that it's safe and desirable to re-send PUT requests if they fail.